modules/schema/src/main/java/org/apache/ignite/internal/schema/catalog/CatalogToSchemaDescriptorConverter.java

```diff
@@ -149,15 +149,15 @@ public static Column convert(int columnOrder, CatalogTableColumnDescriptor colum
      * @param tableDescriptor Descriptor to convert.
      * @return A {@link SchemaDescriptor} object representing the table descriptor.
      */
-    public static SchemaDescriptor convert(CatalogTableDescriptor tableDescriptor) {
+    public static SchemaDescriptor convert(CatalogTableDescriptor tableDescriptor, int tableVersion) {
         Set<String> keyColumnsNames = Set.copyOf(tableDescriptor.primaryKeyColumns());
 
         List<Column> keyCols = new ArrayList<>(keyColumnsNames.size());
         List<Column> valCols = new ArrayList<>(tableDescriptor.columns().size() - keyColumnsNames.size());
 
         int idx = 0;
 
-        for (CatalogTableColumnDescriptor column : tableDescriptor.columns()) {
+        for (CatalogTableColumnDescriptor column : tableDescriptor.schemaVersions().get(tableVersion).columns()) {
```

Comment:
`get()` might return `null`. Let's explicitly assert that it's not `null` if we are sure this should never happen.

Reply:
Ok